panels/config/config-entries/ha-config-entries.html

```diff
@@ -0,0 +1,177 @@
+<link rel='import' href='../../../bower_components/paper-button/paper-button.html'>
+<link rel='import' href='../../../bower_components/paper-card/paper-card.html'>
+<link rel='import' href='../../../bower_components/iron-flex-layout/iron-flex-layout-classes.html'>
+
+<link rel='import' href='../../../src/util/hass-mixins.html'>
+<link rel='import' href='../../../src/resources/ha-style.html'>
+<link rel='import' href='../../../src/layouts/hass-subpage.html'>
+
+<link rel='import' href='./ha-config-flow.html'>
+
+<dom-module id="ha-config-entries">
+  <template>
+    <style include='iron-flex ha-style'>
+      .content {
+        padding: 8px;
+      }
+      .entries {
+        margin: 0 -4px;
+      }
+      .entries paper-card {
+        margin: 4px;
+      }
+    </style>
+    <hass-subpage title='Integrations'>
+      <div class='content'>
+        <h1>Configured</h1>
+        <template is='dom-if' if='[[!_entries.length]]'>
+          <i>Nothing configured.</i>
+        </template>
+        <div class='entries layout horizontal wrap'>
+          <template is='dom-repeat' items='[[_entries]]'>
+            <paper-card heading='[[item.title]]'>
+              <div class='card-content'>
+                Integration: [[item.domain]]<br>
+                State: [[item.state]]<br>
+                Added by: [[item.source]]
+              </div>
+              <div class="card-actions">
+                <paper-button on-tap='_removeEntry'>REMOVE</paper-button>
+              </div>
+            </paper-card>
+          </template>
+        </div>
+
+        <template is='dom-if' if='[[_progress.length]]'>
+          <h1>To be finished</h1>
+          <div class='entries layout horizontal wrap'>
+            <template is='dom-repeat' items='[[_progress]]'>
+              <paper-card heading='[[item.domain]]'>
+                <div class="card-actions">
+                  <paper-button on-tap='_continueFlow'>Configure</paper-button>
+                </div>
+              </paper-card>
+            </template>
+          </div>
+        </template>
+
+        <h1>Add integration</h1>
+        <div class='entries layout horizontal wrap'>
+          <template is='dom-repeat' items='[[handlers]]'>
+            <paper-card heading='[[item]]'>
+              <!-- <div class="card-content">Some content</div> -->
+              <div class="card-actions">
+                <paper-button on-tap='_createFlow'>Configure</paper-button>
+              </div>
+            </paper-card>
+          </template>
+        </div>
+      </div>
+    </hass-subpage>
+
+    <ha-config-flow
+      hass='[[hass]]'
+      flow-id='[[flowId]]'
+      on-flow-closed='_flowClose'
+    ></ha-config-flow>
+  </template>
+</dom-module>
+
+<script>
+{
+  class HaConfigManager extends window.hassMixins.NavigateMixin(Polymer.Element) {
+    static get is() { return 'ha-config-entries'; }
+
+    static get properties() {
+      return {
+        hass: Object,
+        isWide: Boolean,
```

Comment:
can be removed from all files

Reply:
It's used by `ha-config-section` but is indeed not used here, removed it 👍